prisma/seeds/seed.ts

```diff
@@ -62,6 +61,72 @@ async function publishingStatusMigration() {
   console.log(`${templatesToMigrate.length} were migrated for Publishing Status`);
 }
 
+async function templateSchemaMigration() {
+  const templates = await prisma.template.findMany({
+    select: {
+      id: true,
+      jsonConfig: true,
+    },
+  });
+
+  const templatesToMigrate = templates
+    .filter((template) => (template.jsonConfig as Record<string, unknown>).submission !== undefined)
+    .map((template) => {
+      const formProperties = template.jsonConfig as AnyObject;
+
+      const newJsonConfiguration = {
+        titleEn: formProperties.form.titleEn ?? "",
+        titleFr: formProperties.form.titleFr ?? "",
+        internalTitle: formProperties.internalTitleEn ?? "",
```

Comment:
We had a [discussion](https://gcdigital.slack.com/archives/C03TW5SRZT2/p1673887957329939) on Slack last week where it was not unsure whether we wanted to have a single `internalTitle` instead of two `internalTitleEn` / `internalTitleFr`. 
I mentioned the fact that I would keep that property in the JSON for now until we are sure that one simple internal title is enough. Probably easier to change if this is in the JSON vs Database fields.

Reply:
Confirmed yesterday that one 'internalTitle' field is enough.  We're treating it more like a file name then an attribute.

Reply:
Yes please.  This way on the MyForms page we don't need to pull all of the Template info/json from the DB by only the internal title.